client/data/multi-currency/selectors.js

```diff
@@ -17,7 +17,7 @@ const getMultiCurrencyState = ( state ) => {
 };
 
 export const getCurrencies = ( state ) => {
-	return getMultiCurrencyState( state ).currencies || [];
+	return getMultiCurrencyState( state ).currencies || {};
```

Comment:
Did you double check that this won't break the MC settings UI?

Reply:
It shouldn't as the actual data structure is an object. But I'll check that!

Reply:
It works because both will result in `undefined` when trying to access a key that doesn't exist.

```js
const a = [];
const b = {};

a.available // undefined
b.available // undefined
```

I'll keep the change because it makes more sense to be an object than an array. Is that ok for you?